### DISCUSSION TITLE
Transaction stuck

### DISCUSSION THREAD

**Discussion by vlad-adsd** [Chia-Network/chia-blockchain#16725] [Support]
After sending to CHIA wallet my transaction did not arrive and it has Success - Unspent status. How can I fix it? When will the coins arrive?
Txn Hash
0xae8471e9841781d8034913bc2810fc051b35c7443fb72f96aa99dd147f1d2cd0
I would be grateful for any support. 
Thank you,
Vlad

**Comment by BrandtH22** [comment]
Her @vlad-adsd , as a heads up we can generally provide more timely, thorough support in our discord server (https://discord.gg/chia).

What I recommend trying is a resync of the wallet:

- If using the gui go to settings -> advanced -> resync . This will close the client and upon reopening it will resync.
- If using the cli then I would stop chia, rename the wallet db folder in ~\.chia\mainnet\wallet\db then restart chia.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
